content/doc/upgrade-guide/2.164.adoc

```diff
@@ -0,0 +1,20 @@
+---
+layout: documentation
+title:  Jenkins Upgrade Guide
+notitle: true
+---
+
+== Upgrading to Jenkins LTS 2.164.x
+
+Each section covers the upgrade from the previous LTS release, the section on 2.164.1 covers the upgrade from 2.150.3.
+
+=== Upgrading to Jenkins LTS 2.164.1
+
+[java11]
+==== Changes related to Java 11 compatibility
+
+TODO jaxb Plugin explanation (also for Java 8)
```

Comment:
Just link #2101 . But we need to ship it before Wedhesday EoD then

Reply:
https://jenkins.io/doc/administration/requirements/upgrade-java-guidelines/

Reply:
There's a reason that `jaxb` plugin used to depend on core 2.60.x and that reason is not documented anywhere it seems.